operations/configuration_management/Troubleshoot_CFS_Issues.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshoot CFS Issues
+
+Due to CFS' nature as a framework that runs arbitrary Ansible content, there are any number of issues that can arise when attempting to configure a system.
+Many of the issues are transient, especially on larger systems or when a long configuration is applied.
+Because of this, CFS automatically retries configuration in many cases and a single failing session is often not an issue.
+However, when a component is marked as failed or a number of sessions are failing there are two ways to approach debugging.
+
+## Using the CFS Debugger
```

Comment:
If the tool's proper name is CFS Debugger, then capitalizing Debugger here is appropriate. Otherwise Debugger should be lowercase. Please apply or mark resolved, as appropriate.

```suggestion
## Using the CFS debugger
```